packages/syft/src/syft/core/node/new/sqlite_document_store.py

```diff
@@ -312,20 +312,24 @@ class SQLiteStoreClientConfig(StoreClientConfig):
             database, it will be locked until that transaction is committed. Default five seconds.
     """
 
-    filename: Optional[str]
-    path: Optional[Union[str, Path]] = None
+    filename: Optional[str] = None
+    path: Union[str, Path]
     check_same_thread: bool = True
     timeout: int = 5
 
-    @property
-    def temp_path(self) -> str:
-        return tempfile.gettempdir()
+    def __init__(
```

Comment:
I think we could implement this with a pydantic validator instead so we dont need to implement init, but this works!